third_party/move/move-model/src/demove_helper.rs

```diff
@@ -0,0 +1,133 @@
+use std::{collections::BTreeMap, rc::Rc, borrow::Borrow};
```

Comment:
Every source must have copyright.

But I think the file can be eliminated, see below.

Reply:
Need to be addressed